src/platform-includes/sourcemaps/overview/javascript.mdx

```diff
@@ -2,13 +2,19 @@
 
 We've compiled a list of guides on how to upload source maps to Sentry for the most popular JavaScript build tools:
 
+### Sentry Plugin Support
+
 - <PlatformLink to="/sourcemaps/uploading/webpack/">Webpack</PlatformLink>
-- <PlatformLink to="/sourcemaps/uploading/typescript/">
-    TypeScript (tsc)
-  </PlatformLink>
 - <PlatformLink to="/sourcemaps/uploading/rollup/">Rollup</PlatformLink>
 - <PlatformLink to="/sourcemaps/uploading/vite/">Vite</PlatformLink>
 - <PlatformLink to="/sourcemaps/uploading/esbuild/">esbuild</PlatformLink>
+
+### Guides
+
+- <PlatformLink to="/sourcemaps/uploading/typescript/">
+    TypeScript (tsc)
+  </PlatformLink>
+  <Note>If using a bundler and TSC, only the plugin is needed</Note>
```

Comment:
Very vague. People do not understand the words "bundler" and they will no know to what the word "plugin" refers to here.